pom.xml

```
@@ -152,7 +152,7 @@
         <dependency>
             <groupId>com.thoughtworks.xstream</groupId>
             <artifactId>xstream</artifactId>
-            <version>1.4.12</version>
+            <version>1.4.14</version>
```

1.4.15 is now out, which resolves a couple of CVEs:
* http://x-stream.github.io/CVE-2020-26258.html
* http://x-stream.github.io/CVE-2020-26259.html

```suggestion
            <version>1.4.15</version>
```